extensions/objects/templates/image-with-regions-editor.component.spec.ts

```diff
@@ -510,12 +510,12 @@ describe('ImageWithRegionsEditorComponent', () => {
       regionType: 'Rectangle',
       area: [
         [
-          0.04985965130585909,
-          0.0765451050371804
+          0.23225103021579202,
+          0.08157179492961315
```

Comment:
The value inside area[][] here shouldn't have changed from before because of
```typescript
expect(component.yDirectionToggled).toBe(false);
expect(component.xDirectionToggled).toBe(false);
```
See the test just below this one, where 
```typescript
expect(component.yDirectionToggled).toBe(true);
expect(component.xDirectionToggled).toBe(true);
```
Value should change in such case because X and Y direction have been toggled.